web/src/components/tasks/search-business-name/SearchBusinessNameForm.test.tsx

```diff
@@ -118,7 +118,7 @@ describe("<SearchBusinessNameForm />", () => {
   it("shows designator text if name includes the designator", async () => {
```

Comment:
Perhaps you could rename the `"it" statement` so that is has the word error in it. I think it will make it more clear that designator is and error.